portfolio/src/main/webapp/index.html

```diff
@@ -68,26 +68,61 @@ <h1>Makeup Gallery</h1>
       <img src="images/MU8.jpg" alt="A fashion/blogger inspired look" width="650" height="650">
       <img src="images/MU9.JPG" alt="A look inspired by BTS's music video, Euphoria"
        width="650" height="650">
-
-      <h1>Guinea Pig Gallery</h1>
-      <img src="images/GP1.jpg" alt="My guinea pig, Butter, sniffing the camera" width="650" 
-      height="650">
-      <img src="images/GP2.jpg" alt="My guinea pig, Butter, smiling at the camera" width="650" 
-      height="650">
-      <img src="images/GP3.jpg" alt="My guinea pig, Koko, posing for the camera" width="650" 
-      height="650">
-      <img src="images/GP4.jpg" alt="My guinea pig, Taffy, posing for the camera" width="650" 
-      height="650">
-      <img src="images/GP5.jpg" alt="My guinea pig, Butter, running towards the camera" width="650" 
-      height="650">
-      <img src="images/GP6.jpg" alt="My guinea pig, Koko, running towards the camera" width="650" 
-      height="650">
       
     </div>
   </body>
 </html>
 
 
+<!-- Slideshow container -->
+<div class="slideshow-container">
+   <h1>Guinea Pig Gallery</h1>
+   <p>Press the arrows to open and view the Guinea Pig Gallery!</p>
+  <!-- images with number and caption text -->
+  <div class="mySlides fade">
```

Comment:
Use hyphen separated words similar to slideshow-container. It varies place to place, and sometimes code to code. A good rule is to be consistent with whatever code is already present. Also leave off "my" in naming.

You might see something like s/mySlides fade/slides-fade. That means replace "mySlides fade" with "slides-fade"